### DISCUSSION TITLE
Rtl_airband

### DISCUSSION THREAD

**Discussion by amane93** [rtl-airband/RTLSDR-Airband#258] [Q&A]
EDIT: Im on version 3.1.0 i want to install the 3.2.0 one. but even after following the make procedures after extracting the new files it wont change rtl_airband -v shows 3.1.0 version

**Comment by dlpwx** [comment]
You probably are running the latest version. It's a bug. See https://github.com/szpajder/RTLSDR-Airband/issues/199

**Reply by amane93** [reply]
right now i installed the 3.2.0 version after installing ubuntu entirely and the dongle doesnt shut down properly aftr i run the command once. i have to either physically disconnect the usb, or use a usb reset program. reseting usb ports isnt feasible so im open to suggestions

*once i've closed the rtl airband program, the usb keeps heating up.

**Reply by szpajder** [reply]
Whenever the program receives a stopping signal (SIGHUP, SIGINT, SIGQUIT or SIGTERM) it stops all devices before exiting.

Perhaps you are SIGKILLing it?
